frontend/src/employee-frontend/components/FinancePage.tsx

```diff
@@ -18,90 +20,163 @@ import InvoicesPage from './invoices/InvoicesPage'
 import PaymentsPage from './payments/PaymentsPage'
 import VoucherValueDecisionsPage from './voucher-value-decisions/VoucherValueDecisionsPage'
 
+interface PageProps {
+  path: string
+  title: string
+  Page: React.FunctionComponent
+}
+
 export default React.memo(function FinancePage() {
   const { i18n } = useTranslation()
+  const { user } = useContext(UserContext)
 
   const tabs = useMemo(
     () =>
       [
-        {
-          id: 'fee-decisions',
-          link: '/finance/fee-decisions',
-          label: i18n.header.feeDecisions
-        },
-        {
-          id: 'value-decisions',
-          link: '/finance/value-decisions',
-          label: i18n.header.valueDecisions
-        },
-        {
-          id: 'invoices',
-          link: '/finance/invoices',
-          label: i18n.header.invoices
-        },
-        featureFlags.experimental?.voucherUnitPayments
+        user?.permittedGlobalActions.includes('SEARCH_FEE_DECISIONS')
```

Comment:
nullability chaining looks weird here. if user really can be null then `user?.permittedGlobalActions` can also be null so there should be more question marks I think

Reply:
Good point, I'll add the null check.